docs.html

```diff
@@ -81,45 +81,34 @@
 <a href='#json'>%decoding_json</a>
 <a href='#testing'>%testing</a>
 <a href='#memory'>%memory_management</a>
-<a href='#defer'>Defer</a>
-<a href='#orm'>ORM</a>
-<a href='#vfmt'>vfmt</a>
+<a href='#defer'>%defer</a>
```

Comment:
I think `defer` is a keyword, so it doesn't need to be translated to another language

Reply:
I have reverted related change of defer , ORM and vfmt.